corehq/apps/hqwebapp/static/hqwebapp/js/initial_page_data.js

```diff
@@ -94,7 +94,7 @@ hqDefine('hqwebapp/js/initial_page_data', ['jquery', 'underscore'], function ($,
                 throw new Error("URL '" + name + "' not found in registry");
             }
         }
-        return urls[name].replace(/---/g, function () {
+        return urls[name].replace(/(---)|(000)/g, function () {
```

Comment:
I don't follow this - why does it need the placeholder to be digits?

Reply:
Because the URL mapping only accepts digits:
https://github.com/dimagi/commcare-hq/blob/a614c46a6c4398be92b110fcd86a331804a3c53b/corehq/apps/data_interfaces/urls.py#L59-L60